src/graphql-global-id/README.md

```diff
@@ -50,9 +50,13 @@ const fields = {
 };
 ```
 
-# `toGlobalId`
+# Creating opaque ID imperatively (`toGlobalId`)
 
-TKTK
+You can you `toGlobalId` to create the unique and opaque identifier manually. However, try to always use the `GlobalID` type constructor since it handles the type automatically. It's usually a bad practice to use `toGlobalId` in your types unles you really need it. Usage:
```

Comment:
`You can you `toGlobalId`` -> `You can use`?

Reply:
`unles` -> `unless`

Reply:
thanks